server/src/main/java/org/opensearch/action/support/clustermanager/TransportClusterManagerNodeAction.java

```diff
@@ -372,4 +443,14 @@ protected String getMasterActionName(DiscoveryNode node) {
         return getClusterManagerActionName(node);
     }
 
+    /**
+     * Determines if transport action needs to check local cluster-state term with manager before
+     * executing the action on manager. This is generally true for actions that are read-only and can be executed locally
+     * on node if the term matches with cluster-manager.
+     * @return - true to perform term check and then execute the action
+     */
+    protected boolean checkTermVersion() {
+        return false;
```

Comment:
why the default is false?

Reply:
`TransportClusterManagerNodeAction` is being extended both by Read and Write actions. Hence the default is false and will be overrided to true for all Read actions in future (`TransportClusterManagerNodeReadAction`).